Fix schedule instructions for "prefect deploy" (#12101)
PrefectHQ/prefect: docs/concepts/schedules.md

Schedules tell the Prefect API how to create new flow runs for you automatically on a specified cadence.

> You can add a schedule to any flow [deployment](/concepts/deployments/).

Reviewer: ```suggestion
You can add a schedule to any [deployment](/concepts/deployments/).
```

The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.

!!! tip "Support for multiple schedules"
    We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.

>     Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and [worker-based deployments](/concepts/work-pools/) with `prefect deploy` will arrive soon.

Reviewer: Serve support PR has at least one approval, so likely to be in this release, too. #12107

Author: Oh, good point!

Author: I'm going to wait and we can update that if we do indeed ship `flow.serve` support, which is very likely given the PR... but not merged yet.

- Via a the `cron`, `interval`, or `rrule` parameters if building your deployment via the [`serve` method](/concepts/flows/#serving-a-flow) of the `Flow` object or [the `serve` utility](/concepts/flows/#serving-multiple-flows-at-once) for managing multiple flows simultaneously
- If using [worker-based deployments](/concepts/work-pools/)
  - Through the interactive `prefect deploy` command
  - With the `deployments` -> `schedule` section of the `prefect.yaml` file )

## Support for multiple schedules

> We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to Deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.

Reviewer: ```suggestion
We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.
```

> Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and `prefect deploy` will arrive in Q1 2024.

Reviewer: ```suggestion
Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and `prefect deploy` will arrive soon.
```

## Creating schedules through the UI

> You can add a schedule on the **Deployments** tab of the UI.

Reviewer: ```suggestion
You can add schedules on a **Deployment** page in the UI.
```

> Under **Schedule** select the **Add** button.

Reviewer: ```suggestion
Under **Schedules** select the **+ Schedule** button.
```

> If you don't see the **Schedule** section, expand the three dot menu on the top right of the page.

Reviewer: ```suggestion
If you don't see the **Schedule** section, increase the width of the page.
```

Then select **Interval** or **Cron** to create a schedule.

![Prefect UI with Interval button selected](/img/ui/interval-schedule.png)

Once a schedule has been created, a number of scheduled flow runs will be visible in the UI.
The schedule is viewable in human-friendly text on the **Deployments** page.

> You can edit the schedule by selecting the **Edit** button on the **Deployment** page.

Reviewer: ```suggestion
You can edit a schedule by selecting **Edit** from the three dot menu next to a schedule on a **Deployment** page.
```
